actionview/test/javascript_package_test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class JavascriptPackageTest < ActiveSupport::TestCase
+  def test_compiled_code_is_in_sync_with_source_code
```

Comment:
Does this test actually check both files correctly? Yarn build will update both files after the first run and no difference will be detected in the second file since potential changes are already saved.

Reply:
Thanks, that's right.
I updated test in 4ce13308a28596fcc9d97f94087cd75a4144be0c .